ml-agents/mlagents/trainers/learn.py

```diff
@@ -105,6 +105,9 @@ def main():
     for i in range(num_runs):
         if seed == -1:
             run_seed = np.random.randint(0, 10000)
-        p = multiprocessing.Process(target=run_training, args=(i, run_seed, options))
+        q = Queue()
```

Comment:
I think there should be some inline comments on what this does. 
Also, `q` only ever contains one True boolean or nothing, maybe there is a better object that that we could use?

Reply:
I think this can be a `multiprocessing.Event` (https://docs.python.org/2/library/multiprocessing.html#multiprocessing.Event).

Then I'd call this something like `training_started_event` so it's more self-documenting.